### DISCUSSION TITLE
Call Me MAYbe? ☎️  [Pull Requests Mini Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#55875] [Pull Requests]
Hello? Is it me you’re looking for? May is for our [maintainers](https://maintainermonth.github.com/), but it also brings new releases to the world of Pull Requests and shoutouts to some community members. Definitely pick up the phone for our updates below.  

⚡**Pull Requests Praise**: Thanks to these fine folks for keeping our Community awesome ✔️
* @bbkx226: For a thorough answer to a [question](https://github.com/orgs/community/discussions/53225) about PR templates in multiple repositories. They even quickly came back to answer a follow-up question. Kudos to you @bbkx226 ⭐! 
* @urig-40seas: Gave some [amazing feedback in a discussion](https://github.com/orgs/community/discussions/54551#discussioncomment-5864724) on secure private attachments and worked alongside our team to highlight areas of improvement. Nicely done! 

ℹ️ **What’s new**: The latest Pull Requests updates
* **More secure private attachments** 📎: Previously, all attached (drag-and-dropped) images and videos on GitHub Issues, Pull Requests, Discussions, and wikis were available to view without authentication if you knew their direct URL. Now, future attachments associated with private repositories can only be viewed after logging in. [View the changelog](https://github.blog/changelog/2023-05-09-more-secure-private-attachments/) or [give feedback](https://github.com/orgs/community/discussions/54551). 
* **Pull request merge queue (public beta): API support and recent fixes**: [Learn more](https://github.blog/changelog/2023-04-19-pull-request-merge-queue-public-beta-api-support-and-recent-fixes/) about our recent updates to the merge queue. 

📣**Announcements**: The latest updates in the GitHub Universe 🆕
* **Addressing GitHub’s recent availability issues**: GitHub recently experienced several availability incidents, both long running and shorter duration. We have since mitigated these incidents and all systems are now operating normally. [Read on through this link](https://github.blog/2023-05-16-addressing-githubs-recent-availability-issues/) for more details about what caused these incidents and what we’re doing to mitigate in the future.
* **Game Bytes · May 2023** 🎮: Game Bytes is our monthly series taking a peek at the world of gamedev on GitHub—featuring game engine updates, game jam details, open source games, mods, maps, and more. [Game on](https://github.blog/2023-05-10-game-bytes-may-2023/)!

I’m hanging up the phone on Pull Requests for this month - but hope to see you in June.

**Comment by bbkx226** [comment]
@queenofcorgis Thank you for tagging me! It means a lot to be recognized by such a prestigious platform.

**Reply by queenofcorgis** [reply]
You're so welcome. Thank you for being such a wonderful community member - keep it up!

**Comment by queenofcorgis** [comment]
Catch up with what's new in Pull Requests, follow the linked text [here](https://github.com/orgs/community/discussions/58811) for our new check-in for June!
